database/seeds/DatabaseSeeder.php

```diff
@@ -14,10 +14,8 @@ public function run()
     {
         Model::unguard();
 
-        //$this->call(MagazineTableSeeder::class);
         $this->call(CategoryTableSeeder::class);
         $this->call(October16NewCategoryTableSeeder::class);
-        //$this->call(August16ArticleSeeder::class);
```

Comment:
These changes are not related to login form.

This probably happened due to wrong branch creation/merges.